specifications/move_adapter/README.md

```diff
@@ -967,128 +837,126 @@ the code cache.
 
 ### Publishing
 
-Clients may publish modules in the system by calling
+Clients may publish modules in the system by calling:
 
 ```rust
-// Publish a Module if passes verification and would be loadable by the VM.
 pub fn publish_module(
-    &self,
-    // module to publish
+    &mut self,
     module: Vec<u8>,
-    // publisher
     sender: AccountAddress,
-    // data store
-    data_store: &mut dyn DataStore,
-    // amount of computation allowed
     cost_strategy: &mut CostStrategy,
+    log_context: &impl LogContext,
 ) -> VMResult<()>;
 ```
 
-the module is in a [serialized form](#Binary-Format) and the VM performs the
+The `module` is in a [serialized form](#Binary-Format) and the VM performs the
 following steps:
 
-* **Deserialize the module**. If the module does not deserialize, an error is
+* Deserialize the module: If the module does not deserialize, an error is
 returned with a proper `StatusCode`.
 
-* **Check that the module address and the `sender` address are the same**. This
+* Check that the module address and the `sender` address are the same: This
```

Comment:
Not sure how important it is so feel free to ignore. This isn't the case under closed module publishing; Diem Root publishes modules with the `0x1` address and this is special-cased in the publishing logic.

Reply:
I'm inclined to pass on updating this one right now. I'm pretty sure the next bullet is also outdated as well. The VM section of this doc still needs work....